__tests__/app.test.js

```diff
@@ -20,14 +20,29 @@ describe('GET /api/topics task 3',() => {
                     expect.objectContaining({
                       slug: expect.any(String),
                       description: expect.any(String),
-                    })
-                  );
-                // expect.objectContaining({
-                //     'slug': expect(typeof(topicObj['slug'])).toBe('string'),
-                //     'description': expect(typeof(topicObj['description'])).toBe('string')
-                // })
-                // expect(Object.keys(obj)).toMatchObject(['description','slug'])
-            })
+                    }));
+            });
         });
     });
 });
+
+
+//4
+describe.only('GET /api/articles task 4',() => {
+    test('responds with status code 200, appropriate object properties and values',() => {
+        return request(app).get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            expect(Array.isArray(body.articles)).toBe(true);
+            body.articles.forEach((articleObj) => {
+                expect(articleObj).toEqual(
+                    expect.objectContaining({
+                        author: expect.any(String),
+                        title: expect.any(String),
+                        article_id: expect.any(Number),
+                        topic: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        comment_count: expect.any(String)
+                    }))
+            });
```

Comment:
We also need a test here to check that the results are sorted in descending order by date